cfme/tests/infrastructure/test_timelines.py

```diff
@@ -63,15 +66,19 @@ def gen_events(delete_fx_provider_event, provider, test_vm):
     mgmt.start_vm(test_vm.name)
 
 
-def count_events(vm_name, nav_step):
+def count_events(vm, nav_step):
```

Comment:
I think we should think about moving count_events into the vm object. What do you think?